modules/fishing.py

```diff
@@ -30,29 +24,37 @@
 shark = emoji.emojize(':shark:')  # 98 - 100 100d
 
 
-class FishingSession:
-    id: int
-    state: int
+class FishingState(Enum):
+    WAITING_FOR_BITE = 1
+    FISH_BITING = 2
+    HOOKING = 3
```

Comment:
This is what I had in mind for states.

After some converting, I ended up removing "Not fishing" because we can just use the lack of state to mean "not fishing". Further explanation is below.